pages/wizard/rhev/setup_type.py

```diff
@@ -0,0 +1,124 @@
+from selenium.webdriver.common.by import By
+from pages.base import Base
+
+from pages.wizard.regions.deployment_step_bar import DeploymentStepBar
+
+class SetupType(Base):
+    _page_title = "Setup Type"
+
+    # locators
+    _self_hosted_radio_loc = (
+        By.XPATH,
+        '//input[@type = "radio" and @value = "selfhost"]'
+    )
+    _hypervisor_engine_radio_loc = (
+        By.XPATH,
+        '//input[@type = "radio" and @value = "rhevhost"]'
+    )
+    _cancel_loc = (
+        By.XPATH,
+        '//button[contains(@class,"btn") and contains(., "Cancel")]'
+    )
+    _back_loc = (
+        By.XPATH,
+        '//a[contains(@class, "btn") and contains(., "Back")]'
+    )
+    _next_loc = (
+        By.XPATH,
+        '//button[contains(@class,"btn") and contains(., "Next")]'
+    )
+
+    # These locators for the cancel model frame.   Note since the code
+    # is always there we don't need to select the dialogue as its just
+    # a frame in the existing page that sometimes is visible.
+    _exit_and_delete_loc = (
+        By.XPATH,
+        '//button[contains(.,"Exit and Delete")]'
+    )
+    _exit_and_save_loc = (
+        By.XPATH,
+        '//button[contains(.,"Exit and Save")]'
+    )
+    _continue_working_loc = (
+        By.XPATH,
+        '//button[contains(.,"Continue Working")]'
+    )
+
+    # properties
+    @property
+    def self_hosted_btn(self):
+        return self.selenium.find_element(*self._self_hosted_radio_loc)
+
+    @property
+    def hypervisor_engine_btn(self):
+        return self.selenium.find_element(*self._hypervisor_engine_radio_loc)
+
+    def cancelBtn(self):
```

Comment:
For the cancelBtn, backBtn, and nextBtn methods, I think these should follow the snake_case naming convention that the other methods in the file use. Also, the cancelBtn method needs an `@property` decorator